src/kiwixapp.cpp

```diff
@@ -544,6 +544,11 @@ void KiwixApp::saveWindowState()
   mp_session->setValue("windowState", getMainWindow()->saveState());
 }
 
+void KiwixApp::saveVoiceName(const QString& langName, const QString& voiceName)
+{
+  mp_session->setValue(langName + "/voice", voiceName);
```

Comment:
This way of generating the setting name from variable data introduces a possibility of conflict with another setting with a name generated in a different way. I think it is better to at least start setting names with a fixed known prefix (either move "voice" to the beginning, or prepend "tts:" or "text2speech:")

Reply:
Done